src/lwc/rd2ElevateCreditCardForm/rd2ElevateCreditCardForm.html

```diff
@@ -28,7 +27,7 @@ <h1 class='slds-text-body_regular' data-qa-locator='heading'>{labels.elevateWidg
                     </lightning-layout-item>
 
 
-                    <lightning-layout-item size="12" multiple-rows="true" class='slds-p-horizontal_small slds-p-top_small'>
```

Comment:
I'm curious why the need to remove multiple-rows here, and if this necessitates a deeper QA role on this work item then if it were just jest tests?

Reply:
multiple-rows is not supported on lightning-layout-item.  It was causing a parse error during jest test execution.  It should never have been there.

Reply:
Thanks